conf/nginx/rate-limiting.conf.jnj

```diff
@@ -49,24 +49,27 @@ limit_req_zone $http_authorization zone=staticauth:10m rate=30r/s;
 limit_req_zone $request_id zone=staticauth:10m rate=300r/s;
 {% endif %}
 
-limit_req_zone $http1_bucket zone=dynamicauth_very_light_http1:10m rate=60r/s;
-limit_req_zone $http2_bucket zone=dynamicauth_very_light_http2:10m rate=600r/s;
-limit_req_zone $namespaced_http1_bucket zone=namespaced_dynamicauth_very_light_http1:10m rate=60r/s;
-limit_req_zone $namespaced_http2_bucket zone=namespaced_dynamicauth_very_light_http2:10m rate=600r/s;
+# Rate limits for the v2 API endpoint.
+limit_req_zone $http1_bucket zone=v2_endpoint_http1:10m rate={{ http1_bucket[0] }}r/s;
```

Comment:
consider using a dict like `http1_bucket['v2']`

Reply:
Done in new commit.